scripts/sondes.yaml

```diff
@@ -4865,4 +4857,4 @@
 - flag: UGLY
   launch_time: 2020-02-11 09:48:23
   platform: P3
-  sonde_id: P3-0211_s44
+  sonde_id: P3-0211_s44
```

Comment:
There's a new line missing at the end of the file.

Reply:
Oops.